core/src/labels/labelMesh.cpp

```diff
@@ -13,4 +19,102 @@ void LabelMesh::addLabel(std::unique_ptr<Label> _label) {
     m_labels.push_back(std::move(_label));
 }
 
+void LabelMesh::loadQuadIndices() {
+    if (s_quadGeneration == s_validGeneration) {
+        glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, s_quadIndexBuffer);
+
+    } else {
+        s_quadGeneration = s_validGeneration;
+
+        std::vector<GLushort> indices;
+        indices.reserve(maxVertices / 4 * 6);
+
+        for (size_t i = 0; i < maxVertices; i += 4) {
```

Comment:
I don't see how this could be nicely done, moving this to the constructor would introduce concurrency, maybe statically initialize it? I just noticed that it can take up to 1ms + the buffer upload on this part that's why I was worried about it, but since it's happening once it shouldn't be noticed.